### PR TITLE
DCOS_OSS-5672: 3 fixes to dcos_install.ps1

### PR DESCRIPTION
**Pull request by sergiimatusEPAM**
## High-level description

dcos_install.ps1 enhancements and bug-fixes:
- removed RunOnce logic
- updated cluster.conf params
- added Set-EnvPath function for properly handle Path variable, in idempotent way 
- downloaded detect_ip*ps1 scripts

## Corresponding DC/OS tickets (required)

  - [DCOS_OSS-5672](https://jira.mesosphere.com/browse/DCOS_OSS-5672) Remove RunOnce.ps1, start winpanda directly from dcos_install.ps1 w/o reboot
  - [DCOS_OSS-5677](https://jira.mesosphere.com/browse/DCOS_OSS-5677) Fix cluster config generation
  - [DCOS_OSS-5673](https://jira.mesosphere.com/browse/DCOS_OSS-5673) added Set-EnvPath function for properly handle Path variable, in idempotent way
## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot changelog-not-required minor fixes

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot request review @jongiddy

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot request review @DominikDary

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot request review @vadymzagorodko

File: gen/build_deploy/powershell/dcos_install.ps1 [hunk 1]
```diff
@@ -249,11 +270,12 @@ function main($url, $version, $masters) {
...
     Write-Log("Local IP: $($local_ip)")
-    $content = "$($masternodecontent)`n[distribution-storage]`nRootUrl=$($bootstrap_url)`nPkgRepoPath=$($version)/genconf_win/serve`nPkgListPath=latest.package_list.json`n[local]`nLocalPrivateIPAddr=$($local_ip)"
+    $content = "$($masternodecontent)`n[distribution-storage]`nRootUrl=$($bootstrap_url)`nPkgRepoPath=$($version)/genconf_win/serve/packages`nPkgListPath=$($version)/genconf_win/serve/package_lists/latest.package_list.json`n[local]`nLocalPrivateIPAddr=$($local_ip)"
     CreateWriteFile "$($basedir)\conf" "cluster.conf" $content
 
-    # Creating RunOnce scheduled task along with the RunOnce.ps1 script
-    CreateRunOnceScheduledTask $basedir $masters $local_ip
+    Write-Log("Running Winpanda.py ...")
+    & python C:\winpanda\bin\winpanda.py setup;
+    & python C:\winpanda\bin\winpanda.py start;
```
Comment (DominikDary) [inline gen/build_deploy/powershell/dcos_install.ps1]: This looks like a bug, why is winpanda in c:\?

Comment (sergiimatusEPAM) [reply gen/build_deploy/powershell/dcos_install.ps1]: it's not a bug, it's a feature - winpanda unpacked into C:\
 @anatolii-borysov requested such some time back

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot label Ready For Review

File: gen/build_deploy/powershell/dcos_install.ps1 (hunk 1 above)
Comment (DominikDary) [reply gen/build_deploy/powershell/dcos_install.ps1]: We had an offline discussion and for the demo I'm fine using this version. 
But after that we need to adjust to the adjustments @jongiddy requested.

**Pull request merged**